<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetObjForm</class>
 <widget class="QWidget" name="NetObjForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NetObj Monitor Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QWidget" name="wNetCmd" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_12">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <widget class="QLabel" name="lbClientID">
            <property name="text">
             <string>ClientID:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sbClientID">
            <property name="minimum">
             <number>-999999999</number>
            </property>
            <property name="maximum">
             <number>999999999</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <widget class="QLabel" name="lbEvent">
            <property name="text">
             <string>Event:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbEvent"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <widget class="QLabel" name="lbObj_UID">
            <property name="text">
             <string>Obj UID:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sbObj_UID">
            <property name="minimum">
             <number>-999999999</number>
            </property>
            <property name="maximum">
             <number>999999999</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <widget class="QLabel" name="lbPropName">
            <property name="text">
             <string>PropName:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lePropName"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <widget class="QLabel" name="lbExtCmdData">
            <property name="text">
             <string>ExtCmdData:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leExtCmdData"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btnSendNetCmd">
          <property name="text">
           <string>Send
NetCmd</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnAdd_NetObj">
           <property name="text">
            <string>Add NetObj</string>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
           </property>
           <property name="shortcut">
            <string>Ctrl+Ins</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDel_NetObj">
           <property name="text">
            <string>Del NetObj</string>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/del.png</normaloff>:/images/del.png</iconset>
           </property>
           <property name="shortcut">
            <string>Ctrl+Del</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lbJSON">
           <property name="text">
            <string>JSON:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSave_Obj">
           <property name="text">
            <string>Save Obj</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLoad_Obj">
           <property name="text">
            <string>Load Obj</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSave_Root">
           <property name="text">
            <string>Save Root</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLoad_Root">
           <property name="text">
            <string>Load Root</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeView" name="tvNetObj">
         <property name="styleSheet">
          <string notr="true">QTreeView::item:focus
{
border : 2px solid blue;
/*color: white;*/
selection-color: black;
/*selection-background-color: red;*/
}
</string>
         </property>
         <property name="tabKeyNavigation">
          <bool>true</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="indentation">
          <number>25</number>
         </property>
         <attribute name="headerMinimumSectionSize">
          <number>140</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QScrollArea" name="saNetObj">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <widget class="QWidget" name="saNetObj_WidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>315</width>
         <height>478</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
